### PR TITLE
Cards style

### PR DESCRIPTION
**Pull request by AlexandreMig**
Created pages for each donation
Designed css
Hard-coded values

### CODE REVIEW

File: src/main/webapp/style.css [hunk 1]
```diff
@@ -7,3 +7,57 @@
...
+  align-items: center;
+  background: yellow;
+  border-radius: 16px;
+  display: flex;
+  font-size: 14px;
+  justify-content: center;
+  margin: 24px 0;
+  padding: 8px;
+  width: 100px;
+}
```
Comment (OrlandoFabian) [inline src/main/webapp/style.css]: I would say that maybe you should create a separate file that says something like "card_style.css" and add this code to that new file.

File: src/main/webapp/Clothing.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+          <div class="listing-chat">
+              <a href="">Chat</a>
+          </div>
+          
+        </div>
+      </div>
+    </div>
+
+  </body>
+</html>
```
Comment (OrlandoFabian) [inline src/main/webapp/Clothing.html:38]: Spacing and indentation. We need to decide if we want 4 space indentation or 2 space.

File: src/main/webapp/Food.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+          </div>
+
+          <div class="listing-chat">
+              <a href="">Chat</a>
+          </div>
+          
+        </div>
+      </div>
+    </div>
+
```
Comment (OrlandoFabian) [inline src/main/webapp/Food.html:37]: Spacing and indentation.

File: src/main/webapp/General.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+          <div class="listing-chat">
+              <a href="">Chat</a>
+          </div>
+          
+        </div>
+      </div>
+    </div>
+
+  </body>
+</html>
```
Comment (OrlandoFabian) [inline src/main/webapp/General.html:38]: Spacing and indentation.

File: src/main/webapp/Sanitary.html [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+          <div class="listing-chat">
+              <a href="">Chat</a>
+          </div>
+          
+        </div>
+      </div>
+    </div>
+
+  </body>
+</html>
```
Comment (OrlandoFabian) [inline src/main/webapp/Sanitary.html:39]: Spacing and indentation.

File: src/main/webapp/Clothing.html (hunk 2 above)
Comment (AlexandreMig) [reply src/main/webapp/Clothing.html:38]: I did 2 spaces for all of them

File: src/main/webapp/cards.css [hunk 6]
```diff
@@ -0,0 +1,64 @@
+#content {
+  margin-left: auto;
+  margin-right: auto;
+  width: 650px;
+}
+
+#greeting-container {
+  margin-top: 20px;
+}
+
```
Comment (mcshen-sps) [inline src/main/webapp/cards.css]: If you aren't using these rules, there's no reason to include them in the stylesheet

File: src/main/webapp/Clothing.html (hunk 2 above)
Comment (mcshen-sps) [reply src/main/webapp/Clothing.html:38]: Make sure you chat with the team and come to an agreement so we don't have inconsistent spacing in sections that different people have worked on

**Review by mcshen-sps — APPROVED** [review]
Good work Alexandre! In the future, try using `git rebase ...` instead of `git pull ...` to incorporate new changes from main into your development branch. Then those new changes will show as the original code instead of as new changes in the pull request UI

**Pull request merged**